src/shared/components/FluxMonacoEditor.scss

```diff
@@ -48,29 +48,40 @@
   position: relative;
   overflow: hidden;
 
-  .sync-bar {
-    position: absolute;
-    width: 48px;
+  .composition-sync--on {
+    background: #00a3ff26; // blue with 15% opacity
   }
 
-  .sync-bar--on {
-    background: #00a3ff26; // blue with 15% opacity
+  .composition-sync--on--first {
     border-top: 1px solid $c-pool;
-    border-bottom: 1px solid $c-pool;
+    // composition sync icon
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M4.82511 4.0427L7.10622 1.6971L5.67243 0.302734L0.0902793 6.0427H16V4.0427H4.82511ZM11.1749 11.9571H0V9.95714H15.9097L10.3275 15.6971L8.89376 14.3027L11.1749 11.9571Z' fill='%2300a3ff'/%3E%3C/svg%3E%0A");
+    background-repeat: no-repeat;
+    background-size: 11px 11px;
+    background-position: $cf-space-3xs $cf-space-3xs;
+  }
 
-    .sync-icon {
-      color: $c-pool;
-    }
+  .composition-sync--on--last {
+    border-bottom: 1px solid $c-pool;
   }
 
-  .sync-bar--off {
+  .composition-sync--off {
     opacity: 50%;
     background-color: $cf-grey-35;
+  }
+
+  .composition-sync--off--first {
+    opacity: 50%;
     border-top: 1px solid $cf-white;
-    border-bottom: 1px solid $cf-white;
+    // composition sync icon
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M4.82511 4.0427L7.10622 1.6971L5.67243 0.302734L0.0902793 6.0427H16V4.0427H4.82511ZM11.1749 11.9571H0V9.95714H15.9097L10.3275 15.6971L8.89376 14.3027L11.1749 11.9571Z' fill='white'/%3E%3C/svg%3E%0A");
```

Comment:
I got this svg from our [Figma design](https://www.figma.com/file/0qAntPk5LVangAHguWT74X/Query-Experience-Project?node-id=1824%3A79393) and encoded using [URL-encoder for SVG](https://yoksel.github.io/url-encoder/)